guides/user/canary/stage/index.md

```diff
@@ -0,0 +1,181 @@
+against the target environment.
+---
+layout: single
+title:  "Add a Canary stage to a pipeline"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+
+If you have [enabled canary for your application](/setup/canary/) and have one
+or more [canary configs prepared](), you can now add a canary stage to your
+pipeline and configure it to perform canary analysis for your deployment,
+
+This stage type is for the canary _analysis_ only. The canary stage doesn't
+perform any provisioning or cleanup operations for you. Those must be configured
+elsewhere in your pipeline
+
+## About the canary stage
+
+The canary analysis can be performed over data points collected beginning from
+the moment of execution and into the future, or it can be performed over a
+specified time interval.
+
+### Real-time versus retrospective analysis
+
+A real-time analysis means that the canary analysis is performed over a time
+interval beginning at the moment of execution. In a retrospective analysis the
+canary analysis is performed over an explicitly specified time interval (likely
+in the past).
+
+### Metric scope
+
+Metric scope is the where and when for your canary analysis. It describes the
+specific baseline and canary server groups, the start and end times and interval,
+and the cloud resource on which the baseline and canary are running.
+
+You can also refine the scope using [extended params](#extended-params).
+
+## Configure the canary stage
+
+<!-- something here about where in the pipeline you'd put the canary stage, and
+other permutations, like multiple stages? -->
+
+1. In the pipeline in which you will run the canary, click __Add stage__.
+
+   This pipeline needs to be in an application that has access to the [canary
+   configuration](/guides/user/canary/config/) you want to use.
+
+1. For __Type__ select __Canary__.
+
+1. Give the stage a name, and use the __Depends On__ field to position the stage
+downstream of its dependencies.
+
+   ![Canary stage declaration](/guides/user/canary/stage/canary_stage_top.png)
+
+1. Select the __Analysis Type__&mdash;either __Real Time__ or __Retrospective__.
+
+   * __Real Time__
+
+     The analysis happens for a specified time period, beginning when the stage
+     executes (or after a specified __Delay__).
+
+     For __Real Time__, also specify the number of hours to run (__Lifetime__).
```

Comment:
@dorbin @duftler 
https://screenshot.googleplex.com/DvSehyiJ7Ur

Both the docs and UI show the time to run the canary stage in hours.

Is this a hard limitation? Best practice? 
Developers sometimes want very fast pipelines, there - a minimal short canary can still provide value (say 10-20 minutes  to collect data and judge).

Is that supported?

Reply:
@danielpeach 
@duftler 
Do we/should we support fractional hours here?

Reply:
Just verified that fractions of hours don't work, but I think it's probably reasonable to support.

Reply:
I agree we should support this. Will take a bit of work (checked and it is an int specifying the number of hours in the backend, as opposed to a `Duration`), but we should work on lifting this constraint.

Reply:
https://github.com/spinnaker/kayenta/issues/283

Reply:
Where in kayenta is this limited to an int?  I thought the canary scope uses start and end times, so is this a limitation in a specific data source, or in the system as a whole?

Reply:
https://github.com/spinnaker/orca/blob/master/orca-kayenta/src/main/kotlin/com/netflix/spinnaker/orca/kayenta/model/KayentaCanaryContext.kt#L30